test/helpers/istio/install.go

```diff
@@ -108,12 +134,12 @@ func assertIstioInitContainer(t *testing.T, pods corev1.PodList, testDynakube dy
 		istioInitFound := false
 
 		for _, initContainer := range podItem.Spec.InitContainers {
-			if initContainer.Name == istioInitContainerName {
+			if initContainer.Name == istioInitName {
```

Comment:
we could check with an or clause here for both platforms either openshift initContainerName || kubernetes initContainer name, so we can save the platform switch

Reply:
`istioInitName` used somewhere else in the function, so I just moved the logic to decide which init name to look for into `determineIstioInitContainerName`